test/models/movies_test.js

```diff
@@ -50,9 +49,31 @@ describe('Movie', function() {
       });
     });
   });
+
+  describe('#findBy', function() {
+    // because of how we seeded the db, this also tests that it will only return exact title matches
+    it('returns 1 movie where the name is Jaws', function(done) {
+      movie.findBy('title', 'Jaws', function(err, rows) {
+        assert.equal(err, undefined);
+        assert.equal(rows.length, 1);
+        done();
```

Comment:
Can we add assert.equal(rows[0].title, 'Jaws'); since we are saying that is the name that should be returned in the it statement?